front/playwright.config.ts

```diff
@@ -14,7 +14,7 @@ const config: PlaywrightTestConfig = {
   testDir: './tests',
 
   /* Maximum time one test can run for. */
```

Comment:
Is it in second ? If yes, can you add this precision in the comment ?

Reply:
Done